mw-webapp/src/logic/waysTable/renderLinkInCell/renderLinkInCell.module.scss

```diff
@@ -0,0 +1,6 @@
+@import "src/styles/variables.scss";
+
+.link {
+  color: $lightblue;
```

Comment:
I think better to use words primary, secobdary and etc for naming of color vars. It will help us in the future to change styles/ What do you think?

Reply:
I think we can switch to primary\secondary after we have a design ready, right now it's hard to decide which color should be what